tests/commands/create/test_cleanup_app_content.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+
+from ...utils import create_file
+
+
+@pytest.fixture
+def myapp_unrolled(myapp, support_path, app_packages_path_index):
+    # Create some files that can be cleaned up
+    create_file(support_path / "dir1" / "a_file1.txt", "pork")
+    create_file(support_path / "dir1" / "a_file2.doc", "ham")
+    create_file(support_path / "dir1" / "b_file.txt", "eggs")
+    create_file(support_path / "dir2" / "b_file.txt", "spam")
+    create_file(support_path / "other" / "b_file.txt", "wigs")
+
+    return myapp
+
+
+def test_no_cleanup(create_command, myapp_unrolled, support_path):
+    """If there are no cleanup directives, bundle content isn't touched."""
+
+    # Cleanup app content
+    create_command.cleanup_app_content(myapp_unrolled)
+
+    # Confirm the files are still there
+    assert (support_path / "dir1" / "a_file1.txt").exists()
+    assert (support_path / "dir1" / "a_file2.doc").exists()
+    assert (support_path / "dir1" / "b_file.txt").exists()
+    assert (support_path / "dir2" / "b_file.txt").exists()
+    assert (support_path / "other" / "b_file.txt").exists()
+
+
+def test_dir_cleanup(create_command, myapp_unrolled, support_path):
+    """A directory can be cleaned up."""
+    myapp_unrolled.cleanup_paths = ["path/to/support/dir1"]
+
+    # Cleanup app content
+    create_command.cleanup_app_content(myapp_unrolled)
+
+    # Confirm the files in dir1 have been removed
+    assert not (support_path / "dir1" / "a_file1.txt").exists()
+    assert not (support_path / "dir1" / "a_file2.doc").exists()
+    assert not (support_path / "dir1" / "b_file.txt").exists()
```

Comment:
This should also check whether the directory itself has been removed. And if it the directory doesn't exist, then the checks for the individual files are unnecessary.

We should also test a configuration that removes every file in a directory, but leaves the directory itself.